src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php

```diff
@@ -922,11 +949,20 @@ public function hasCast($key, $types = null)
      */
     public function getCasts()
     {
-        if ($this->getIncrementing()) {
-            return array_merge([$this->getKeyName() => $this->getKeyType()], $this->casts);
-        }
+        return $this->getIncrementing()
+            ? array_merge([$this->getKeyName() => $this->getKeyType()], $this->casts)
+            : $this->casts;
+    }
```

Comment:
This seems noise to me, the functionality did not change, just the code was re-ordered for personal preference, am I right?

If so, this shouldn't be in this PR.

Reply:
Returned